### PR TITLE
docs(textfield): refactored the controls regarding  prefix/suffix

### PR DESCRIPTION
**Pull request by mJarsater**
**Describe pull-request**  
Refactored the controls regarding  prefix/suffix.

**Solving issue**  
Fixes: [AB#2676](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2676)

**How to test**  
1. Go to storybook
2. Check in Components -> Textfield
3. Test the controls for the suffix/prefix.

**Screenshots**  
-

**Additional context**  
-

### CODE REVIEW

**Comment by mJarsater** [comment]
Question for everyone, should we hide the suffix if readonly is true?

File: tegel/src/components/textfield/textfield.stories.ts [hunk 1]
```diff
@@ -191,17 +199,32 @@ const Template = ({
...
+        </span>`
+            : ''
+        }
+        </sdds-textfield>
   </div>
   `,
   );
 };
 
+// ${true ? '<sdds-icon name="cross" slot="sdds-prefix"></sdds-icon>' : ''}
```
Comment (theJohnnyMe) [inline tegel/src/components/textfield/textfield.stories.ts:226]: I guess this can be removed.

**Review by theJohnnyMe — CHANGES_REQUESTED** [review]

File: tegel/src/components/textfield/textfield.stories.ts (hunk 1 above)
Comment (mJarsater) [reply tegel/src/components/textfield/textfield.stories.ts:226]: Woops! Absolutely 😄

Comment (mJarsater) [inline tegel/src/components/textfield/textfield.stories.ts:226]:
```suggestion
```

**Pull request closed without merging**